keyboards/gray_studio/hb85/readme.md

```diff
@@ -2,59 +2,47 @@
 
 ![Gray Studio HB85](https://i.imgur.com/Sk3cupR.jpg)
 
-A HitBit MSX Computer inspired Keyboard.
+A Sony HitBit MSX Computer inspired Keyboard.
 
 Keyboard Maintainer: [Felipe Coury](https://github.com/fcoury)  
```

Comment:
@mechmerlin you are welcome to make yourself the maintainer if you want, since you did most of the job for this keyboard.

Reply:
lol thanks. However, this is a rather unique board that I actually don't own. I'll keep it as is =)